graphql/resolvers.go

```diff
@@ -124,6 +124,34 @@ func (r *queryResolver) MyPublicKeys(ctx context.Context) ([]*restModel.APIPubKe
 	return publicKeys, nil
 }
 
+func (r *taskResolver) AbortInfo(ctx context.Context, at *restModel.APITask) (*AbortInfo, error) {
+	if at.Aborted != true {
+		return nil, nil
+	}
+
+	info := AbortInfo{
+		User:   &at.AbortInfo.User,
+		TaskID: &at.AbortInfo.TaskID,
+	}
+
+	abortedTask, err := task.FindOne(task.ById(at.AbortInfo.TaskID))
```

Comment:
You can simplify this to `task.FindOneId()` https://github.com/evergreen-ci/evergreen/blob/05d7b78d23cd59007ec5378783ab5aa65fbb704b/model/task/db.go#L814